### ISSUE TITLE
[Bug] Microphone doesn't work

### ISSUE THREAD

**Issue by zarEclEC** [23rd/tdesktop#52]
### Steps to reproduce
Settings => Advanced => Call settings

I have only an usb microphone

### Expected behaviour
It should work

### Actual behaviour
It doesn't.
![immagine](https://user-images.githubusercontent.com/6240983/128403882-cee99188-4425-4656-b523-d5eac2715884.png)
As you can see it seems like it's off

### Configuration
**Operating system:** Win 10 21H1 (build 19043.1151)

**Version of Telegram Desktop:** 2.9

**Used theme**: Dark

### Other infos

I tested on telegram official and it works there

**Comment by Devocub** [comment]
When I go to call settings to check mic it doesn't work, also it doesn't work during calls. It works in v 2.8.13. It doesn't work since v2.9.0

```[ALSOFT] (II) Got device "OpenAL Soft on Microphone (3- USB PnP Sound Device)", "{4905455D-916F-42BD-96FC-F020152FD704}", "{0.0.1.00000000}.{4905455d-916f-42bd-96fc-f020152fd704}"
[ALSOFT] (II) Capture format: Mono, Int16, 48000hz, 12000 / 12000 buffer
[ALSOFT] (II) Got message "Open Device" (0x0000, this=0F5F9488)
[ALSOFT] (II) Got message "Reset Device" (0x0001, this=0F5F9488)
[ALSOFT] (II) Requesting capture format:
    FormatTag      = 0xfffe
    Channels       = 1
    SamplesPerSec  = 48000
    AvgBytesPerSec = 96000
    BlockAlign     = 2
    BitsPerSample  = 16
    Size           = 22
    Samples        = 16
    ChannelMask    = 0x4
    SubFormat      = {00000001-0000-0010-8000-00aa00389b71}
[ALSOFT] (II) Got capture format:
    FormatTag      = 0xfffe
    Channels       = 1
    SamplesPerSec  = 44100
    AvgBytesPerSec = 176400
    BlockAlign     = 4
    BitsPerSample  = 32
    Size           = 22
    Samples        = 32
    ChannelMask    = 0x4
    SubFormat      = {00000003-0000-0010-8000-00aa00389b71}
[ALSOFT] (II) Created converter for Mono format, dst: Int16 48000hz, src: Float32 44100hz
[ALSOFT] (II) Created capture device 0FBD1EE0, "OpenAL Soft on Microphone (3- USB PnP Sound Device)"
[ALSOFT] (II) Got message "Start Device" (0x0002, this=0F5F9488)
[ALSOFT] (WW) Error generated on device 0FBD1EE0, code 0xa003
[ALSOFT] (WW) Error generated on device 0FBD1EE0, code 0xa003
[ALSOFT] (WW) Error generated on device 0FBD1EE0, code 0xa003
[ALSOFT] (WW) Error generated on device 0FBD1EE0, code 0xa003
[ALSOFT] (WW) Error generated on device 0FBD1EE0, code 0xa003
[ALSOFT] (WW) Error generated on device 0FBD1EE0, code 0xa003
[ALSOFT] (WW) Error generated on device 0FBD1EE0, code 0xa003
keeps repeating```

**Comment by 23rd** [comment]
@Devocub please provide your OS version.

**Comment by Devocub** [comment]
-_- w10 x64 1903

**Comment by 23rd** [comment]
I hope I have fixed that in the 2.9.12 version.

**Comment by zarEclEC** [comment]
I tested and it works
